.github/workflows/main.yml

```diff
@@ -1,61 +1,100 @@
 name: Continuous Integration
 on:
   push:
-    branches: [ master, 1_4_X ]
+    branches: [ master, 1_4_X, wip_bsd_ci ]
   pull_request:
-    branches: [ master, 1_4_X ]
+    branches: [ master, 1_4_X, wip_bsd_ci ]
```

Comment:
obviously this is for POC, and after you've got it working this goes away and we only do it on master.